src/operations/extract.ts

```diff
@@ -0,0 +1,75 @@
+import { IJS } from '../IJS';
+import { Mask } from '../Mask';
+import { assert } from '../utils/assert';
+
+export interface ExtractOptions {
+  /**
+   * Specify from what row the image should be cropped.
```

Comment:
I would rather say that it's the row/column where the beginning of the mask is placed